Makefile

```diff
@@ -4,6 +4,10 @@
 FLASHSIZE:=8mb
 BAUDRATE:=460800
 
+.EXPORT_ALL_VARIABLES:
+
+BINDGEN_EXTRA_CLANG_ARGS="-D CONFIG_I2C_BUS_BACKWARD_CONFIG"
```

Comment:
This appears to be unavoidable at the moment. 

It would be great it if was something we can set in the module so the downstream user doesn't need to think about it, but haven't found a way to either
- set the env var from the dependency
- get embuild to recognize this as a [`CInclArgs`](https://docs.rs/embuild/latest/embuild/build/struct.CInclArgs.html).

if setting this in our template's `.cargo/config.toml` is harmless, it might be something to consider until we:
- have support for the newer i2c driver or
- figure out how to get this into the embuild system without stepping on esp-idf-sys's use of it